draft-ietf-quic-recovery.md

```diff
@@ -302,11 +302,10 @@ Retransmission Timeout mechanisms.
 ### Crypto Handshake Timeout
 
 Data in CRYPTO frames is critical to QUIC transport and crypto negotiation, so a
-more aggressive timeout is used to retransmit it.  Below, the term "handshake
-packet" is used to refer to packets containing CRYPTO frames, not packets with
-the specific long header packet type Handshake.
+more aggressive timeout is used to retransmit it.  Below, the term "crypto
+packet" is used to refer to packets containing CRYPTO frames.
 
-The initial handshake timeout SHOULD be set to twice the initial RTT.
+The initial crypto timeout SHOULD be set to twice the initial RTT.
```

Comment:
Can we rename this?  I initially assumed that the handshake would be abandoned when this fires.  It's a crypto retransmission timeout (a term you already use).

Reply:
I went with consistently using crypto handshake timeout, because I think it reads well and is consistent, but I can change it to something else if you prefer.